web_widget_radio_tree/static/src/js/widget.js

```diff
@@ -0,0 +1,90 @@
+openerp.web_widget_radio_tree = function (instance) {
+
+    var QWeb = instance.web.qweb;
+
+    instance.web.list.columns.add('field.radio_tree', 'instance.web.list.RadioTreeColumn');
+
+    instance.web.list.RadioTreeColumn = instance.web.list.Column.extend({
+        _format: function (row_data, options) {
+            var name = options.model + '_' + this.id;
+            return _.template(
+                '<input type="radio" name="<%-name%>" <%-checked%> readonly="readonly"></input>', {
+                    name: name,
+                    checked: row_data[this.id].value ? 'checked' : '',
+                });
+        }
+    });
+
+    instance.web.form.widgets.add('radio_tree', 'instance.web.form.RadioTree');
+
+    instance.web.form.RadioTree = instance.web.form.FieldBoolean.extend({
+        template: 'RadioTree',
+        start: function() {
+            var self = this;
+            this.$checkbox = $('input', this.$el);
+            var radio_name = this.__parentedParent.model + '_' + this.$checkbox[0].name;
+            this.$checkbox.attr('name', radio_name);
+            this.setupFocus(this.$checkbox);
+            this.$el.click(_.bind(function() {
```

Comment:
use https://github.com/OCA/OCB/blob/8.0/addons/web/static/src/js/openerpframework.js#L404 and  https://github.com/OCA/OCB/blob/8.0/addons/web/static/src/js/openerpframework.js#L805